target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -115,8 +114,13 @@ case "$FIRMWARE" in
 		/usr/bin/fritz_cal_extract -i 1 -s 0x3D000 -e 0x212 -l 12064 -o /lib/firmware/$FIRMWARE $(find_mtd_chardev "urlader0") || \
 		/usr/bin/fritz_cal_extract -i 1 -s 0x3D000 -e 0x212 -l 12064 -o /lib/firmware/$FIRMWARE $(find_mtd_chardev "urlader1")
 		;;
+	linksys,ea8300)
+		ath10kcal_extract "ART" 36864 12064
+		# OEM assigns 4 sequential MACs
+		ath10kcal_patch_mac_crc $(macaddr_setbit_la $(macaddr_add "$(cat /sys/class/net/eth0/address)" 4))
+		;;
 	openmesh,a62)
-		ath10kcal_extract "0:ART" 36864 12064
```

Comment:
Please don't change/break other boards (since this would also require a DTS update in qcom-ipq4019-a62.dts). But this is outside the scope of this patch.